src/Monolog/Formatter/DatadogFormatter.php

```diff
@@ -1,35 +0,0 @@
-<?php
-
-namespace sgoettsch\MonologDatadog\Formatter;
-
-use Monolog\Formatter\JsonFormatter;
-use Monolog\Level;
-use Monolog\Logger;
-use Monolog\LogRecord;
-use stdClass;
-
-class DatadogFormatter extends JsonFormatter
-{
-    protected bool $includeStacktraces = true;
-
-    public function format(LogRecord $record): string
-    {
-        $normalized = $this->normalize($record);
-
-        if (isset($normalized['context']) && $normalized['context'] === []) {
-            $normalized['context'] = new stdClass;
-        }
```

Comment:
`LogRecord` does not have a writable `context` property. The context is set on creating the LogRecord object. Converting the empty array to an empty `stdClass` object is not needed anymore.